### PR TITLE
Added a new optional (default enabled) room config

### PR DESCRIPTION
**Pull request by farhatahmad**
## Testing Steps

1- Make sure you run rails db:migrate:with_data
2- Create a new room
3- Make sure the "Allow meeting to be recorded" is enabled by default but can be disabled
4- Play around with room configs and make sure room settings match

### CODE REVIEW

File: app/assets/stylesheets/application.bootstrap.scss [hunk 1]
```diff
@@ -236,11 +236,8 @@ input.search-bar {
 // Custom Select component.
 .select{
   .dropdown-item.active, .dropdown-item:active  {
-    background-color: var(--brand-color) !important;
-  }
-
-  .dropdown-item.active:hover {
-    color: $black !important;
+    background-color: var(--brand-color-light) !important;
+    color: var(--brand-color) !important;
```
Comment (farhatahmad) [inline app/assets/stylesheets/application.bootstrap.scss:240]: Missing styling changes from last PR

File: app/assets/stylesheets/application.bootstrap.scss [hunk 2]
```diff
@@ -381,6 +378,29 @@ input.search-bar {
...
+    &:focus {
+      box-shadow: 0 0 0 0.25rem var(--brand-color-light) !important;
+    }
+  }
+
+  .dropdown-menu {
+    min-width: 260px;
+  }
+}
+
```
Comment (farhatahmad) [inline app/assets/stylesheets/application.bootstrap.scss:403]: Missing styling changes from last PR

File: app/models/room.rb [hunk 3]
```diff
@@ -47,11 +47,14 @@ def get_setting(name:)
...
-    configs = MeetingOption.get_config_value(name: %w[glViewerAccessCode glModeratorAccessCode], provider: user.provider)
-    configs = configs.select { |_k, v| v == 'true' }
+    configs = RoomsConfiguration.joins(:meeting_option).where(provider: user.provider).pluck(:name, :value).to_h
 
     MeetingOption.all.find_each do |option|
-      value = configs.key?(option.name) ? SecureRandom.alphanumeric(6).downcase : option.default_value
+      value = if %w[true default].include? configs[option.name]
+                option.true_value
+              else
+                option.default_value
```
Comment (farhatahmad) [inline app/models/room.rb:56]: default_value is basically the false value

**Comment by scouillard** [comment]
I would change `optional` and `default` for `default_enabled` and `default_disabled`

**Review by alihadi-mazeh — DISMISSED** [review]

**Comment by farhatahmad** [comment]
> I would change `optional` and `default` for `default_enabled` and `default_disabled`

Replaces default with `default_enabled`. Optional can be changed in another PR

**Review by scouillard — APPROVED** [review]

**Pull request merged**